bin/ci

```diff
@@ -1,10 +1,12 @@
 #! /bin/sh
 
-set -e
-
 # https://git.io/vDywe
-export CI_COMMITED_AT=$CI_TIMESTAMP
+export CI_COMMITED_AT=$(date +%s)
+
+set -e
+set -x
 
+rubocop -D
 rake rspec:unit
 # rake rspec:integration
-codeclimate-test-reporter
+# codeclimate-test-reporter
```

Comment:
I've had to disable CodeClimate test coverage reporting because the gem is expecting to access "committed at" from `git` which is not present. The gem has an override to support his on Codeship but not for CircleCI v2.0. :(